js/components/interface/flexLayout2/src/view/TabButton.tsx

```diff
@@ -130,7 +130,13 @@ export class TabButton extends React.Component<ITabButtonProps, any> {
         let leadingContent = undefined;
 
         if (node.getIcon() !== undefined) {
-            leadingContent = <img src={node.getIcon()} />;
+            if (node.getIcon().startsWith('fa ')) {
```

Comment:
Same note here, do we really need to import material in flexlayout when we can achieve this injecting the icons with onRenderTabSet ?

Reply:
Removed